Directory.Build.props

```diff
@@ -1,14 +1,18 @@
 <Project>
-  <Import Condition="Exists('version.props')" Project="version.props" />
-
   <PropertyGroup>
-    <Copyright>Copyright 2008</Copyright>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <PropertyGroup>
+    <Copyright>Copyright 2008-2024</Copyright>
     <Authors>Kurt Schelfthout and contributors</Authors>
     <PackageTags>F# fsharp test random</PackageTags>
     <RepositoryUrl>https://github.com/Fscheck/fscheck</RepositoryUrl>
     <PackageProjectUrl>https://fscheck.github.io/FsCheck/</PackageProjectUrl>
     <PackageLicenseExpression>BSD-3-Clause</PackageLicenseExpression>
     <RepositoryType>git</RepositoryType>
     <DebugType>embedded</DebugType>
+    <PackageIconUrl>https://raw.githubusercontent.com/fscheck/FsCheck/master/docs/files/img/logo.png</PackageIconUrl>
```

Comment:
This is deprecated, but it's what the current NuGet packages (from Paket) have.

Reply:
Don't think that works, if it ever did. in any case I don't see the logo on nuget.org. *shrug*

Reply:
Hmm. In a followup PR I'll just fix this to point at the actual logo (wherever it is).